src/libmongoc/src/mongoc/mongoc-client-side-encryption.c

```diff
@@ -438,6 +446,29 @@ mongoc_client_encryption_encrypt_opts_set_algorithm (
    opts->algorithm = bson_strdup (algorithm);
 }
 
+void
+mongoc_client_encryption_encrypt_opts_set_contention_factor (
+   mongoc_client_encryption_encrypt_opts_t *opts, int64_t contention_factor)
+{
+   if (!opts) {
+      return;
+   }
+   opts->contention_factor.value = contention_factor;
+   opts->contention_factor.set = true;
+}
```

Comment:
Do we care about giving the user a way to "un-set" these options?

Reply:
IMO no. I do not see much of a use case. A `mongoc_client_encryption_encrypt_opts_reset_contention_factor` could be added later if needed.